src/commands/generate/index.ts

```diff
@@ -0,0 +1,67 @@
+import {Command} from '@oclif/core'
+import * as inquirer from 'inquirer'
+
+export default class Generator extends Command {
+  static description = 'Generate infrastructure template command'
+
+  static examples = [
+    `$ nimble-infra generate
+    `,
+  ]
+
+  static flags = {}
+
+  static args = []
+
+  async run(): Promise<void> {
+    const questions = [
+      {
+        type: 'list',
+        name: 'platform',
+        message: 'What cloud provider you would like to use?',
+        choices: [
+          {
+            value: 'aws',
+            name: 'AWS',
+          },
+          {
+            value: 'gcp',
+            name: 'GCP',
+          },
+          {
+            value: 'heroku',
+            name: 'Heroku',
+          },
+        ],
+      },
+    ]
+
+    const platformChoice = await inquirer.prompt(questions)
+
+    if (platformChoice.platform === 'aws') {
+      const questions = [
+        {
+          type: 'list',
+          name: 'infrastructureType',
+          message: 'What kind of infrastructure you need?',
```

Comment:
```suggestion
          message: 'What kind of infrastructure do you need?',
```

Reply:
Update the typo here: https://github.com/nimblehq/infrastructure-templates/pull/22/commits/884d8606c4dcaf9b98cf14e1d4f2dad2152a9cef